core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_body_directive.html

```diff
@@ -28,7 +28,7 @@
       </multiple-choice-editor>
 
       <!-- TODO(sll): Update this condition when control of individual field visibility is added. -->
-      <div ng-if="!$last">
+      <div ng-if="!$last" id="getSubfieldId(question, 'Bridge text')">
```

Comment:
ID here is simple HTML attribute and it is used in `SimpleEditorSidebarDirective` to scroll to it